src/main/java/org/alliancegenome/curation_api/jobs/executors/HTPExpressionDatasetAnnotationExecutor.java

```diff
@@ -49,40 +54,63 @@
 			BackendBulkDataProvider dataProvider = BackendBulkDataProvider.valueOf(fms.getFmsDataSubType());
 
 			bulkLoadFileDAO.merge(bulkLoadFileHistory.getBulkLoadFile());
-			
-			List<Long> datasetIdsLoaded = new ArrayList<>();
+
+			List<Long> htpAnnotationsIdsLoaded = new ArrayList<>();
+			List<Long> previousIds = htpExpressionDatasetAnnotationService.getAnnotationIdsByDataProvider(dataProvider.name());
 			
 			bulkLoadFileHistory.setTotalRecords((long) htpExpressionDatasetData.getData().size());
+			boolean success = runLoad(bulkLoadFileHistory, dataProvider, htpExpressionDatasetData.getData(), htpAnnotationsIdsLoaded);
+			if(success) {
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' is not followed by whitespace.
<!-- __reviewdog__:ChAyYmFjNjY5NzUwNzA3NTZiEgpjaGVja3N0eWxl -->